packages/target-graph/tests/prioritize.test.ts

```diff
@@ -58,8 +66,8 @@ describe("prioritize", () => {
         targets.set(to, createTarget({ packageName: packageName!, task, dependencies: [], dependents: [], priority: 1 }));
       }
 
-      targets.get(from)!.dependencies.push(to);
```

Comment:
The bug here was that this was backwards so the tests were not capturing the correct behavior.